### PR TITLE
Add how to on creating namespace in UK cluster

### PR DESCRIPTION
**Pull request by vinn946**

### CODE REVIEW

File: how-to-docs/namespace-how-to.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### How do I create a namespace in the UK cluster?
+Please visit the [ACP-Kube-Namespaces](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube-namespaces) repo on Gitlab to create a namespace.
+
+In order to request a namespace for creation, clone and branch the repository and create a namespace definition within the relevant cluster directory with the namespace name as a .yaml file. The name of the file must match the label defined in the yaml.
+
++ Clusters are defined by the folders within the `namespaces/` directory. Within this directory lists all defined namespaces for each given cluster.
+
```
Comment (Joseph-Irving) [inline how-to-docs/namespace-how-to.md]: This directory contains all defined namespaces for each given cluster.

File: how-to-docs/namespace-how-to.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### How do I create a namespace in the UK cluster?
+Please visit the [ACP-Kube-Namespaces](https://gitlab.digital.homeoffice.gov.uk/Devops/acp-kube-namespaces) repo on Gitlab to create a namespace.
+
+In order to request a namespace for creation, clone and branch the repository and create a namespace definition within the relevant cluster directory with the namespace name as a .yaml file. The name of the file must match the label defined in the yaml.
+
```
Comment (Joseph-Irving) [inline how-to-docs/namespace-how-to.md]: This kind of just says what's further down. Not sure if this is needed

**Comment by vinn946** [comment]
Shall i also change the how-to-docs readme to include a link to this and sarah-jane's local cert file?

**Comment by Joseph-Irving** [comment]
yeah that would be useful @vinn946

File: how-to-docs/namespace-how-to.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+metadata:
+  labels:
+    name: foo
+  name: foo
+```
+to be defined in `namespaces/bar/foo.yaml`
+- The name of the file must match the label defined in the yaml.
+
++ Clone and branch the ACP-Kube-Namespaces repo and create your namespace file
++ Submit a PR of your branch to master for your namespace to be created.
```
Comment (Joseph-Irving) [inline how-to-docs/namespace-how-to.md]: Minor thing: should probably just clarify that once your PR is accepted your namespace is created, LG otherwise!

**Pull request merged**